src/PerfView/Triggers.cs

```diff
@@ -189,9 +189,11 @@ public override void Dispose()
         {
             m_monitoringDone = true;
 
+#if PERFVIEW
```

Comment:
Outside of TraceEvent, we should not need the PERFVIEW define.   These defines date to a time when the code was shared with Visual Studio, but we don't do that anymore.    Thus I don't want to add more (I would like to get rid of it).     Thus I would like to know what forced you to use it here.   With luck we can simply not have it.